app/src/main/java/com/android/sample/ui/authentication/SignUp.kt

```diff
@@ -272,14 +232,7 @@ fun LinkSpotifyButton() {
               Text(
                   modifier = Modifier.testTag("linkText"),
                   text = "Link",
-                  style =
-                      TextStyle(
-                          fontSize = 14.sp,
-                          lineHeight = 20.sp,
-                          fontFamily = FontFamily(Font(R.font.roboto)),
-                          fontWeight = FontWeight(500),
-                          letterSpacing = 0.14.sp,
-                          color = PrimaryPurple))
+                  style = TypographyBeatLink.labelSmall)
```

Comment:
Consider utilizing MaterialTheme.typography to further centralize theme control and maintain consistency throughout the application.

```suggestion
                  style = MaterialTheme.typography.labelSmall)
```